access/two_factor_authentication.md

```diff
@@ -13,15 +13,15 @@ This policy applies to all Lullabot employees and contractors.
 
 ### Explanation and Implementation
 
-Traditional authentication consists of a username and password. However, there is a more secure way to authenticate, called two-factor authentication. Two-factor authentication consists of something you know (a username and password) combined with a code available from something you have. The "something you have" is usually another device, like your phone.
+Traditional authentication consists of a username and password. However, there is a more secure way to authenticate, called two-factor authentication. Two-factor authentication consists of something you know (a username and password) combined with a code available from something you have. The "something you have" is usually a single use PIN.
```

Comment:
"usually" here motivated me to change to ... what we are usually using these days.